pyproject.toml

```diff
@@ -31,6 +32,9 @@ dev = [
     "ruff>=0.8.1",
 ]
 
+[tool.uv]
+trusted-publishing = "always"
```

Comment:
i think the default is automatic, doesn't that default to trusted when in ci?

Reply:
Yes, this is a debugging artefact, from trying to figure out why the trusted-publishing didn't work. 
Removing this.